evalml/tests/model_understanding_tests/prediction_explanations_tests/test_algorithms.py

```diff
@@ -105,8 +105,9 @@ def test_shap(estimator, problem_type, n_points_to_explain, X_y_binary, X_y_mult
     if problem_type not in estimator.supported_problem_types:
         pytest.skip("Skipping because estimator and pipeline are not compatible.")
 
-    if problem_type == ProblemTypes.MULTICLASS and estimator.model_family == ModelFamily.CATBOOST:
-        pytest.skip("Skipping Catboost for multiclass problems.")
+    if problem_type == ProblemTypes.MULTICLASS:
+        if estimator.model_family == ModelFamily.CATBOOST:
```

Comment:
Is there a reason why we split this check?

Reply:
Sorry, that's right.  I split it when I was still skipping for KNN.  I'll put it back!